packages/plugin-ext/src/main/browser/menus/menus-contribution-handler.ts

```diff
@@ -404,7 +404,7 @@ export class MenusContributionPointHandler {
         if (!action.command) {
             return Disposable.NULL;
         }
-        const selectedRepository = () => this.toScmArgs(this.scmService.selectedRepository);
+        const selectedRepository = () => this.toScmArg(this.scmService.selectedRepository);
```

Comment:
@ChayaLau can you describe why this change is necessary, I didn't see any information in the commit that would explain how it fixes or relates to multi-root.

Reply:
> @ChayaLau can you describe why this change is necessary, I didn't see any information in the commit that would explain how it fixes or relates to multi-root.

sure,
the function `toScmArgs` returns an array type of the argument it receives, the function `toScmArg` returns an object.
the value is passed to the `ScmExtImpl` class and registered to the process arguments for the ScmExtImpl 
```
commands.registerArgumentProcessor({
            // eslint-disable-next-line @typescript-eslint/no-explicit-any
            processArgument: (arg: any) => {
```
there is expects the argument as a single object, and returns as it is an array type and not recognised as desired sourse control :
`if (!ScmCommandArg.is(arg)) {
                    return arg;
                }`

when i sent it as a single object the issue was solved and the source control was found and returned.